cli/template/page-studs/index/with-auth-trpc-tw.tsx

```diff
@@ -86,29 +68,162 @@ const AuthShowcase: React.FC = () => {
+          clip-rule="evenodd"
+          fill="#fff"
+          fill-rule="evenodd"
+        ></path>
+      </svg>
+    ),
+  },
+
+  {
+    name: "Prisma",
+    description:
+      "Prisma is the best way to work with databases in your NextJS app. It provides a simple, type-safe API to query your database, and it can be used with a bunch of different databases, including Postgres, MySQL, SQL Server, and SQLite.",
+    documentation: "https://www.prisma.io/docs/concepts",
+    icon: (
+      <svg
+        role="img"
+        viewBox="0 0 24 24"
+        xmlns="http://www.w3.org/2000/svg"
+        className="h-10 fill-white py-1"
+      >
+        <title>Prisma</title>
+        <path d="M21.8068 18.2848L13.5528.7565c-.207-.4382-.639-.7273-1.1286-.7541-.5023-.0293-.9523.213-1.2062.6253L2.266 15.1271c-.2773.4518-.2718 1.0091.0158 1.4555l4.3759 6.7786c.2608.4046.7127.6388 1.1823.6388.1332 0 .267-.0188.3987-.0577l12.7019-3.7568c.3891-.1151.7072-.3904.8737-.7553s.1633-.7828-.0075-1.1454zm-1.8481.7519L9.1814 22.2242c-.3292.0975-.6448-.1873-.5756-.5194l3.8501-18.4386c.072-.3448.5486-.3996.699-.0803l7.1288 15.138c.1344.2856-.019.6224-.325.7128z"></path>
+      </svg>
+    ),
+  },
+  {
+    name: "TailwindCSS",
+    description:
+      "TailwindCSS is a utility-first CSS framework that helps you build beautiful, responsive designs without any extra configuration. It’s built with utility-first principles, and is completely customizable and extendable.",
+    documentation: "https://tailwindcss.com/docs",
+    icon: (
+      <svg
+        role="img"
+        viewBox="0 0 24 24"
+        xmlns="http://www.w3.org/2000/svg"
+        className="h-10 aspect-square fill-[#06B6D4] py-1"
+      >
+        <title>TailwindCSS</title>
+        <path d="M12.001,4.8c-3.2,0-5.2,1.6-6,4.8c1.2-1.6,2.6-2.2,4.2-1.8c0.913,0.228,1.565,0.89,2.288,1.624 C13.666,10.618,15.027,12,18.001,12c3.2,0,5.2-1.6,6-4.8c-1.2,1.6-2.6,2.2-4.2,1.8c-0.913-0.228-1.565-0.89-2.288-1.624 C16.337,6.182,14.976,4.8,12.001,4.8z M6.001,12c-3.2,0-5.2,1.6-6,4.8c1.2-1.6,2.6-2.2,4.2-1.8c0.913,0.228,1.565,0.89,2.288,1.624 c1.177,1.194,2.538,2.576,5.512,2.576c3.2,0,5.2-1.6,6-4.8c-1.2,1.6-2.6,2.2-4.2,1.8c-0.913-0.228-1.565-0.89-2.288-1.624 C10.337,13.382,8.976,12,6.001,12z"></path>
+      </svg>
+    ),
+  },
+  {
+    name: "NextAuth.js",
+    description:
+      "When you need a flexible, secure, and scalable authentication solution, NextAuth.js is the go-to. It ties into your existing database and provides a simple API to manage users and sessions, and ties into NextJS and tRPC to provide a seamless experience.",
+    documentation: "https://next-auth.js.org/getting-started/introduction",
+    icon: (
+      <Image
+        src="/nextauth.png"
+        className="h-10 w-8 py-1"
+        alt="NextAuth.js"
+        width="100"
+        height="100"
+      />
+    ),
+  },
+];
```

Comment:
Can we try minimizing it a bit? I like having similar theming on the two but I think this is too many lines to scroll when you wanna rip it out ;)

Reply:
yeah i think you're right, trimmed it by 80 lines :)